addons/mail/static/src/core/common/message_reaction_menu.js

```diff
@@ -14,7 +14,7 @@ import { Dialog } from "@web/core/dialog/dialog";
 import { useService } from "@web/core/utils/hooks";
 
 export class MessageReactionMenu extends Component {
-    static props = ["close", "message"];
+    static props = ["close", "message", "currentReaction?"];
```

Comment:
`currentReaction` => `reaction`

This is more a preference, but it's worth using shorter names when we can. This helps quite a lot to reduce complexity of the code, as most of our time in dev is reading code.